dnngior/build_model.py

```diff
@@ -46,74 +46,77 @@ def refine_model(gpfilledModel, draftModel=None, scale = 1000, unscalled = None)
+
+                metab.compartment = metabSufix
+
+                if compounds_dict[metabName]['formula'] is None:
+                    compounds_dict[metabName]['formula']= ''
+
+                metab.elements = compounds_helper.parseFormula(compounds_dict[metabName]['formula'])
+
+                metab.formula = compounds_helper.buildFormula(compounds_helper.parseFormula(compounds_dict[metabName]['formula']))
+
+                metab.name =  compounds_dict[metabName]['name'] + '_' + metabSufix
             else:
+                #not in the modelSEED database (are specific to GapSeq)
                 metab.annotation = {}
-                
-            metab.charge = compounds_dict[metabName]['charge']
-            
-            metab.compartment = metabSufix
-           
-            if compounds_dict[metabName]['formula'] is None:
-                compounds_dict[metabName]['formula']= ''
-            
-            metab.elements = compounds_helper.parseFormula(compounds_dict[metabName]['formula'])
-            
-            metab.formula = compounds_helper.buildFormula(compounds_helper.parseFormula(compounds_dict[metabName]['formula']))
-            
-            metab.name =  compounds_dict[metabName]['name'] + '_' + metabSufix
-        else:
-            #not in the modelSEED database (are specific to GapSeq)
-            metab.annotation = {}
-            metab.charge = None
-            metab.compartment = metabSufix
-            #metab.elements = ''
-            metab.formula = ''
-            metab.name = metab.id
-    
-    for reac in mc.reactions:
-        reacName = reac.id.split('_')[0]
-        #reacSufix = reac.id.split('_')[1]
-        
-        if reacName in reactions_dict:
-            
-            if reacName in reactions_aliases_dict:
-                reac.annotation = reactions_aliases_dict[reacName]
-                
-            else:
+                metab.charge = None
+                metab.compartment = metabSufix
+                #metab.elements = ''
+                metab.formula = ''
+                metab.name = metab.id
+
+        for reac in mc.reactions:
+            reacName = reac.id.split('_')[0]
+            #reacSufix = reac.id.split('_')[1]
+
+            if reacName in reactions_dict:
+
+                if reacName in reactions_aliases_dict:
+                    reac.annotation = reactions_aliases_dict[reacName]
+
+                else:
+                    reac.annotation = {}
+
+                reac.name = reactions_dict[reacName]['name']
+
+            else:#not in modelSEED
                 reac.annotation = {}
-        
-            reac.name = reactions_dict[reacName]['name']
-        
-        else:#not in modelSEED
-            reac.annotation = {}
-            reac.name = reac.id
-    
+                reac.name = reac.id
+    elif dbType=='BiGG':
+        print("skipping refinement, not currently supported for BiGG models")
```

Comment:
Hmm.. 
I am a bit confused with the BiGG case. 
Would you like to make clear what parts of the dnngior workflow are supported for BiGG models and what are not, so we implement those in the fututre?